src/garage/tf/algos/rl2.py

```diff
@@ -107,8 +116,63 @@ def get_exploration_policy(self):
                 used for meta-RL adaptation.
 
         """
-        return self._policy
+        self._policy.reset()
 
+        class NoResetPolicy:
```

Comment:
can you define this outside RL2 (but still in this file?)

Also, can you move the inner classes to the top of the file, before the RL2 class?

Reply:
Do you mean like `RL2Env` and `RL2Worker` as well? or just `NoResetPolicy` and `RL2AdaptedPolicy`

Reply:
I mean that RL2 should be the last class defined in the file. This allows travelers to read the file from top to bottom without missing anything.